posthog/models/plugin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Plugin(models.Model):
+    name: models.CharField = models.CharField(max_length=200, null=True, blank=True)
+    description: models.TextField = models.TextField(null=True, blank=True)
+    url: models.CharField = models.CharField(max_length=800, null=True, blank=True)
+    config_schema: JSONField = JSONField(default=dict)
+    tag: models.CharField = models.CharField(max_length=200, null=True, blank=True)
+    archive: models.BinaryField = models.BinaryField(blank=True, null=True)
+    from_json: models.BooleanField = models.BooleanField(default=False)
+    from_web: models.BooleanField = models.BooleanField(default=False)
+    error: JSONField = JSONField(default=None, null=True)
+
+
+class PluginConfig(models.Model):
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE, null=True)
+    plugin: models.ForeignKey = models.ForeignKey("Plugin", on_delete=models.CASCADE)
+    enabled: models.BooleanField = models.BooleanField(default=False)
+    order: models.IntegerField = models.IntegerField(null=True, blank=True)
+    config: JSONField = JSONField(default=dict)
+    error: JSONField = JSONField(default=None, null=True)
```

Comment:
I would also add clarification on why we have `error` attributes on both models (i.e. what do we use each for).

Reply:
The error fields are now better documented. I also added them to the interface:

![image](https://user-images.githubusercontent.com/53387/97819347-44d7aa00-1ca8-11eb-8fb6-6d168bc53eda.png)